test/dotnet.Tests/CommandTests/ToolUpdateGlobalOrToolPathCommandTests.cs

```diff
@@ -200,6 +200,20 @@ public void GivenAnExistedLowerversionInstallationWhenUpdateAllItCanPrintSuccess
                 _packageId, LowerPackageVersion, HigherPackageVersion));
         }
 
+        [Fact]
+        public void GivenAnExistedPreviewVersionInstallationWhenUpdateToHigherVersionItSucceeds()
+        {
+            CreateInstallCommand($"-g {_packageId} --version {HigherPreviewPackageVersion} --verbosity minimal");
+            _reporter.Lines.Clear();
```

Comment:
Did you run the test to make sure it failed without your code change?  It looks like you may be skipping executing the install command here.

Reply:
Oh youre right, I totally missed that